R/mutate.R

```diff
@@ -1,9 +1,63 @@
-#' salsa
+#' Mutate new columns
 #'
-#' @param .tbl TODO
-#' @param ... TODO
-#' @param .env TODO
+#' Applies the [ballet()] and makes sure each result have the same [vctrs::vec_size()]
```

Comment:
`dance` uses markdown formatting, in the roxygen, see 
https://cran.r-project.org/web/packages/roxygen2/vignettes/markdown.html

so you can write 

 - `[ballet()]` to create a link to a function in the same 📦 
 - `[vctrs::vec_size()]` to create a link to a function in another package.